tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -546,3 +549,103 @@ def test_positive_virt_card(session, target_sat, module_location, module_org, vm
         target_sat.api.Host(
             id=target_sat.api.Host().search(query={'search': f'name={host_name}'})[0].id
         ).delete()
+
+
+@pytest.mark.e2e
+@pytest.mark.on_premises_provisioning
+@pytest.mark.parametrize('vmware', ['vmware8'], indirect=True)
+@pytest.mark.parametrize('pxe_loader', ['bios'], indirect=True)
+@pytest.mark.parametrize('provision_method', ['bootdisk'])
+@pytest.mark.rhel_ver_match('[8]')
+@pytest.mark.tier3
+def test_positive_provision_end_to_end(
+    module_sca_manifest_org,
+    module_location,
+    pxe_loader,
+    module_vmware_cr,
+    module_vmware_hostgroup,
+    provision_method,
+    vmware,
+    vmwareclient,
+    target_sat,
+    module_provisioning_rhel_content,
+    module_host_template,
+    get_vmware_datastore_summary_string,
+):
+    """Assign Ansible Role to a Hostgroup and verify after provision a Host, job is schedule
+    for ansible role
+
+    :id: 500f32e8-c1db-4ef9-ae20-0dbb5bccf2ea
+
+    :steps:
+        1. Import role(s) available by default.
+        2. Assign role(s) to Hostgroup.
+        3. Provision a Host with Hostgroup.
+        4. In Host -> Overview -> Select the "Recent Job" action.
+        5. Click the Schedule Job button.
+        6. Verify that the Ansible roles has been Scheduled.
+
+    :expectedresults: The role is successfully scheduled with provisioned host.
+
+    :BZ: 2025523
+
+    :customerscenario: true
+    """
+    SELECTED_ROLE = 'theforeman.foreman_scap_client'
+    host_name = gen_string('alpha').lower()
+    guest_os_names = 'Red Hat Enterprise Linux 8 (64 bit)'
+    storage_data = {'storage': {'disks': [{'data_store': get_vmware_datastore_summary_string}]}}
+    network_data = {
+        'network_interfaces': {
+            'nic_type': VMWARE_CONSTANTS['network_interface_name'],
+            'network': f'VLAN {settings.provisioning.vlan_id}',
+        }
+    }
+    target_sat.cli.Settings.set({'name': "remote_execution_connect_by_ip", 'value': "yes"})
```

Comment:
Can we use this instead  `@pytest.mark.parametrize('setting_update', ['remote_execution_connect_by_ip=False'], indirect=True)` ?

Reply:
Also add setting `destroy_vm_on_host_delete` as VM gets delete on vmware